app/furniture/marketplace/tag.rb

```diff
@@ -10,5 +10,10 @@ class Tag < Record
 
     attr_accessor :marketplace
     location(parent: :marketplace)
+
+    positioned
```

Comment:
Shouldn't this positioning be scoped to the `bazaar`? If I'm understanding correctly, this makes the position global across all models in the database, so there can only ever be a tag in position 1 across all of Convene bazaars and marketplaces.